pulp_puppet_plugins/pulp_puppet/plugins/importers/metadata.py

```diff
@@ -10,33 +10,34 @@
 import tempfile
 
 from pulp.common.compat import json
-from pulp.server.exceptions import InvalidValue
+from pulp.server.exceptions import PulpCodedException
 
 from pulp_puppet.common import constants
+from pulp_puppet.plugins import error_codes
 
 
-class ExtractionException(InvalidValue):
+class ExtractionException(PulpCodedException):
```

Comment:
Is this used anywhere in the codebase? If not I think it can be removed from the inheritance tree.